webapp/test/LoginForm.spec.js

```diff
@@ -0,0 +1,275 @@
+import '@testing-library/jest-dom'
+import 'regenerator-runtime'
```

Comment:
Why is this necessary? I think you can configure jest for the same purpose.

Reply:
Could be, this was the simplest way I got asynchronous tests working!